generators/common/templates/sonar-project.properties.ejs

```diff
@@ -0,0 +1,43 @@
+sonar.projectKey=<%= baseName %>
+sonar.projectName=<%= baseName %> generated by jhipster
+sonar.projectVersion=1.0
+
+sonar.sources=<%= MAIN_DIR %>
+sonar.host.url=http://localhost:9001
```

Comment:
I don't know the maven plugin but can you overwrite the host via command line parameter or env variable (as before)?

Reply:
I see no reasons why it can't be done because it's always a maven properties. Environment variable should override it. But If we skip server (using sonar-scanner) I'm not sure how it works.

Reply:
I can do it if you like.

Reply:
every help is welcome ;)
especially with gradle (it could help me to improve myself on gradle).
Also I don't have a lot of time so I'll appreciate

Reply:
@avdev4j Can you still check that you can override the url with something like `mvn -Dsonar.host.url` just to be sure that the property file doesn't take precedence over CLI param. Also this could eventually be configured in the CI-CD subgen.

Reply:
I will merge and check this later.

Reply:
It works  \o/

`./mvnw sonar:sonar -Dsonar.host.url=http://localhost:9012`

```[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.6.0.1398:sonar (default-cli) on project sonar-maven: Unable to execute SonarQube: Fail to get bootstrap index from server: Failed to connect to localhost/0:0:0:0:0:0:0:1:9012: Connection refused (Connection refused)```